src/internet_identity/src/storage/tests.rs

```diff
@@ -80,6 +82,112 @@ fn should_recover_header_from_memory_v7() {
     assert_eq!(storage.version(), 7);
 }
 
+fn add_test_anchor_data<M: Memory + Clone>(storage: &mut Storage<M>, number_of_anchors: usize) {
+    for _ in 0..number_of_anchors {
+        let (anchor_number, mut anchor) = storage
+            .allocate_anchor()
+            .expect("Failure allocating an anchor.");
+        anchor
+            .add_device(sample_device())
```

Comment:
It would be nice to make the devices different (i.e. depend on the index) because this would detect mix-ups in the data.

Reply:
Good idea, done.